api/apps/articles/test/model/index.coffee

```diff
@@ -682,6 +689,22 @@ describe 'Article', ->
         article.keywords.join(',').should.equal 'cool,art,Pablo Picasso,Pablo Picasso,Armory Show 2013,Gagosian Gallery,kana'
         done()
 
+    it 'indexes the article in elasticsearch on save', (done) ->
+      Article.save {
+        author_id: '5086df098523e60002000018'
+        title: 'foo article'
+        published: true
+      }, 'foo', (err, article) ->
+        return done err if err
+        sleep.sleep 1
```

Comment:
i don't know a way around this given that the elasticsearch indexing refresh rate can't go lower than 1 second and receiving an http response is no guarantee that the indexing operation has completed. we have to do this in gravity too for end-to-end integration tests involving elasticsearch.